src/platform/utilities/feature-toggles/helpers.js

```diff
@@ -0,0 +1,26 @@
+const Cookies = require('js-cookie');
+
+// Read Google Analytics ID if available. Otherwise use a random string as a token.
+export function createTokenFromCookie(cookieName) {
+  return Cookies.get(cookieName)
+    ? Cookies.get(cookieName)
+    : Math.random()
+        .toString(36)
+        .substring(2, 15) +
+        Math.random()
+          .toString(36)
+          .substring(2, 15);
+}
+
+// Create cookie containing token
+export function setCookie(cookieName, token) {
+  if (!Cookies.get(cookieName)) {
+    Cookies.set(cookieName, token, { expires: 30 });
+    return token;
+  }
+  return Cookies.get(cookieName);
+}
+
+export function getFlipperId() {
```

Comment:
So the name describes what this function is for, but it might be worth calling out the side-effects in a comment. Namely, that it gets and sets the cookie.